index.html

```diff
@@ -1 +1,15 @@
-
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Lorem Ipsum</title>
+  </head>
+  <body>
+    <main>
+        <h1>Let's see if this works</h1>  
```

Comment:
Could you change the header to "Welcome to vingardium leviosa!" and set the string color to red?